Framework/WorkflowAlgorithms/test/AlignAndFocusPowderTest.h

```diff
@@ -786,9 +786,9 @@ class AlignAndFocusPowderTest : public CxxTest::TestSuite {
     if (m_compressStartTime != "0")
       align_and_focus.setProperty("CompressStartTime", m_compressStartTime);
 
-    // Remove prompt pulse; will cutoff last 6 long-TOF peaks (freq is 200 Hz)
+    // Remove prompt pulse; will cutoff the first peak
     if (m_removePromptPulse)
-      align_and_focus.setProperty("RemovePromptPulseWidth", 1e4);
+      align_and_focus.setProperty("RemovePromptPulseWidth", 2200.0);
```

Comment:
Why are you changing the input parameter?

Reply:
Previously, it worked in a way that in fact all pulses were not filtered. Now  the first pulse which is ended on ~2200 is filtered out. I printed out the date and ensured that the first pulse is filtered out.

Reply:
What is the minimum TOF in the events for this data? My concern is that you greatly reduced the window from 0->10000us to 0->2200us. What is in the range between? An image would be really helpful.

Reply:
![RemovePromptPulseBug](https://github.com/user-attachments/assets/5faab1dc-7e1f-46b5-a12c-9ea6e9d3e86d)
![RemovePromptPulse](https://github.com/user-attachments/assets/b9b2a0f6-7440-490f-8981-602ffe256d36)
Here is a version with the bug, and fixed. Period is 5000 microseconds, and the width is 2200 just to  cut out the first pulse.